04-files.md

```diff
@@ -21,10 +21,11 @@ that finds files whose names match a pattern.
 We provide those patterns as strings:
 the character `*` matches zero or more characters,
 while `?` matches any one character.
-We can use this to get the names of all the html files:
+We can use this to get the names of all the html files.
+We use the `sorted` function to ensure this list is in alphabetical order:
```

Comment:
Please modify to show `glob.glob` first (with items not sorted), then introduce `sorted` in a callout box with a note saying "you should only bother to sort if you really care, and you usually only really care as the last step of processing".